src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -643,5 +662,18 @@ public void AllWellKnownTypeMembers()
                 Assert.NotNull(symbol);
             }
         }
+
+        [Fact]
+        public void FirstTypeAfterExtensionMarker()
+        {
+            var e = WellKnownType.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7_TRest.ExtensionByte();
+            Assert.Equal(1, e);
+        }
+
+        [Fact]
+        public void ExtensionMarkerProperlyAligned()
+        {
+            Assert.Equal(WellKnownType.ExtendedType, WellKnownTypes.GetTypeFromMetadataName("ExtendedType"));
```

Comment:
Since it is not a real type, it shouldn't have a name, I think. Also, I think an assert in static constructor of WellKnownTypes should be sufficient. In any case this test doesn't belong in CSharp tests, it is not a language specific.